packages/vue/src/components/list/MultiDropdownList.jsx

```diff
@@ -383,6 +392,23 @@ const MultiDropdownList = {
 		handleLoadMore() {
 			this.updateQueryOptions(this.$props, true);
 		},
+		getComponent(items, downShiftProps = {}) {
+			const { currentValue } = this.$data;
+			const data = {
+				error: this.error,
+				value: currentValue,
```

Comment:
@ShahAnuj2610 `isLoading` prop is missing here.

Reply:
done